hooks/tk-multi-publish2/ingest/ingest_files.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import pprint
+
+import sgtk
+
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class IngestFilesPlugin(HookBaseClass):
+    """
+    Inherits from PublishFilesPlugin
+    """
+
+    @property
+    def description(self):
+        """
+        Verbose, multi-line description of what the plugin does. This can
+        contain simple html for formatting.
+        """
+
+        return """
+        Ingests the file to the location specified by the templates and
```

Comment:
"...specified by the publish_path_template defined for this item and..."

Reply:
Isn't that producer or artist facing documentation? publish_path_template is inside the templates which we are going to setup anyways.

Reply:
Fair enough, though putting publish_path_template is useful for a TD trying to debug :)

Reply:
Hehe ok, adding it for the TD team :)